e2e/test/ccm_suite_test.go

```diff
@@ -22,7 +22,7 @@ var (
 	useExisting = false
 	reuse       = false
 	clusterName string
-	region      = "us-east"
+	region      = "us-sea"
```

Comment:
I have to change the default region, because Metadata service is currently a dependency for CAPL clusters.